pints/_log_likelihoods.py

```diff
@@ -562,48 +562,51 @@ def __init__(self, problem, sigma):
 
 class MultiplicativeGaussianLogLikelihood(pints.ProblemLogLikelihood):
     r"""
-    Calculates a log-likelihood assuming heteroscedastic Gaussian errors, with
-    the magnitude of the error variance scaling with the problem output.
+    Calculates the log-likelihood for a time-series model assuming a
+    heteroscedastic Gaussian error of the model predictions
+    :math:`f(t, \theta )`. The standard deviation of a heteroscedascic
```

Comment:
The first paragraph of a docstring should usually be a single sentence.

Maybe we could write "...assuming a heteroscedastic (see below) Gaussian error of..." and leave out the second sentence?

Reply:
No problem